nz.mega.MEGAsync.yml

```diff
@@ -0,0 +1,169 @@
+id: nz.mega.MEGAsync
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: '5.15'
+finish-args:
+  - --filesystem=~/MEGA:create
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+command: megasync
+rename-icon: mega
+copy-icon: true
+rename-desktop-file: megasync.desktop
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/man
+modules:
+
+  - name: MEGAsync
+    buildsystem: simple
+    subdir: src
+    build-options:
+      env:
+        DESKTOP_DESTDIR: /app
+    build-commands:
+      - |
+        set -e
+        cd MEGASync/mega
+        ./autogen.sh
+        ./configure \
+          --prefix=/app
+      - qmake
+          PREFIX=/app
+          QMAKE_LIBDIR=/app/lib
+          CONFIG+=FULLREQUIREMENTS
+          MEGA.pro
+      - lrelease MEGASync/MEGASync.pro
+      - make -j${FLATPAK_BUILDER_N_JOBS}
+      - make install
+      - install -D MEGASync/megasync -t /app/bin/
+
+      # Install AppData manifest
+      - install -Dpm0644 ../${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/
+    post-install:
+      # Automatically add build version and date into AppData
+      - |
+        set -e
+        VERSION=$(git describe --tags | sed -e 's/^[vV]//' -e 's/_Linux$//')
+        DATE=$(date -d @$(git show -s --format=%ct) +'%Y-%m-%d')
+        sed \
+          -e "s/@VERSION@/${VERSION}/g" \
+          -e "s/@DATE@/${DATE}/g" \
+          -i /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
```

Comment:
I think renaming  the file to `*.metainfo.xml.in` and generating a new file based on it, would be cleaner.

Reply:
This is definitely what i want to do as well and what should upstreamed.